VirtoCommerce.Platform.Core/Common/ICancellationToken.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.Platform.Core.Common
```

Comment:
Why do we need our own ICancellationToken abstraction?  Why we can't use the default CancellationToken instead?

Reply:
I've added comments inline in code.

Reply:
Thanks. I've looked at this article, but I missed this line  'The order is important, workers will fetch jobs from the critical queue first, and then from the default queue.'